### PR TITLE
Null link fix

### PR DESCRIPTION
**Pull request by pq**
@devoncarew

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.0.1+1
+
+- Fix for when the dart executable can't be found by ``which``.
```
Comment (devoncarew) [inline CHANGELOG.md]: You just need single backticks here :)
```
`which`
```

File: lib/cli_util.dart [hunk 2]
```diff
@@ -37,6 +37,7 @@ Directory getSdkDir([List<String> cliArgs]) {
 
   // Try and locate the VM using 'which'.
   String executable = whichSync('dart', orElse: () => null);
+  if (executable == null) return null;
```
Comment (devoncarew) [inline lib/cli_util.dart]: What about wrapping the code referencing `executable` below in an `if (executable != null)`? That way we can avoid an early return; might prevent issues w/ future code maintenance.

**Comment by devoncarew** [comment]
lgtm! one suggestion for the `if` statement re-work.

File: lib/cli_util.dart (hunk 2 above)
Comment (pq) [inline lib/cli_util.dart]: Sure thing!

**Pull request merged**